multicluster/controllers/multicluster/internal/remote_resourceimport_controller.go

```diff
@@ -209,7 +209,7 @@ func (r *ResourceImportReconciler) handleResImpUpdateForEndpoints(ctx context.Co
 		r.installedResImports.Add(*resImp)
 		return ctrl.Result{}, nil
 	}
-	if _, ok := ep.Labels[common.AntreaMcsAutoGenLabel]; !ok {
+	if _, ok := ep.Labels[common.AntreaMCSAutoGenAnnotation]; !ok {
```

Comment:
Have you done the change to switch from label to annotation, and remove the "multi-cluster" label on exported Services?

Reply:
yes, @jianjuns , this is a multi-cluster service created by our own codes, so I suppose it should be ok to add a label?

Reply:
That is ok, but I think you plan to change from label to annotation?

Reply:
Maybe you like to use label selector for imported services? But you changed the constant name to AntreaMCSAutoGenAnnotation

Reply:
yeah, my mistake, I think both label or annotation should work, I will check and move it as annotation if it can filter event based on annotation.

Reply:
Do you watch with filters? I remember watch can have the apiserver side filter the updates with labels. Not sure about annotation.

Reply:
yes, there is a standalone service_controller.go to do clean up job in case ResourceImport in leader have been removed but member cluster fail to watch the event eg: mcs controller is down accidentally.
controller-runtime provides a  predicate to do the filter. I used it like below in service_controller.go, I think annotation should also work with custom predicate funcs, I will try locally first.
```
func (r *ServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
	instance, _ := predicate.LabelSelectorPredicate(metav1.LabelSelector{
		MatchLabels: map[string]string{
			common.AntreaMCSAutoGenAnnotation: "true",
		},
	})
	return ctrl.NewControllerManagedBy(mgr).
		For(&corev1.Service{}).WithEventFilter(instance).
		WithOptions(controller.Options{
			MaxConcurrentReconciles: common.DefaultWorkerCount,
		}).
		Complete(r)
}
```